### PR TITLE
add AI category

### PR DESCRIPTION
**Pull request by maxandersen**
related to https://github.com/quarkiverse/quarkus-langchain4j/pull/704

@ia3andy the ai category won't be relevant for product nor is the pinned extension in platform so should I remove the pin?

### CODE REVIEW

File: devtools/bom-descriptor-json/src/main/resources/catalog-overrides.json [hunk 1]
```diff
@@ -87,6 +87,16 @@
...
     },
+    {
+      "name": "Artificial Intelligence (AI)",
+      "id": "ai",
+      "description": "Help making Artificial Intgelligence (AI)-infused applications using Large Language Models and more",
+      "metadata": {
+        "pinned": [
+          "io.quarkiverse:quarkus-langchain4j"
+        ]
+      }
```
Comment (maxandersen) [inline devtools/bom-descriptor-json/src/main/resources/catalog-overrides.json:98]:
```suggestion
      
```

**Comment by maxandersen** [comment]
@ia3andy any update here?

**Comment by ia3andy** [comment]
@maxandersen I am not sure what you mean?

**Comment by maxandersen** [comment]
@ia3andy ok to merge ?

**Review by ia3andy — APPROVED** [review]
LGTM

File: devtools/bom-descriptor-json/src/main/resources/catalog-overrides.json [hunk 2]
```diff
@@ -87,6 +87,12 @@
         ]
       }
     },
+    {
+      "name": "Artificial Intelligence (AI)",
+      "id": "ai",
+      "description": "Help making Artificial Intgelligence (AI)-infused applications using Large Language Models and more",
```
Comment (ia3andy) [inline devtools/bom-descriptor-json/src/main/resources/catalog-overrides.json:93]:
```suggestion
      "description": "Help making Artificial Intgelligence (AI)-infused applications using Large Language Models and more"
```

**Review by ia3andy — CHANGES_REQUESTED** [review]
invalid json

**Comment by cescoffier** [comment]
@maxandersen Can you apply the suggestion and produce valid JSON :-)

**Comment by cescoffier** [comment]
@ia3andy Can you do another quick review?

**Review by ia3andy — APPROVED** [review]
LGTM

**Comment by cescoffier** [comment]
Thanks @ia3andy! That was fast (or very reactive ;-))

**Pull request merged**